src/main/webapp/assets/html/swagger-ui.html

```diff
@@ -81,15 +83,15 @@
             <a class="logo" href="#" title="Brooklyn, Version 0.10.0-SNAPSHOT"><!-- Logo added via CSS --></a> <!-- BROOKLYN_VERSION -->
```

Comment:
the line 2 above, don't we need to change `href="/logout"` to `href="../../v1/logout"` ?